docs/_partials/_workshop_prereqs.md

```diff
@@ -0,0 +1,9 @@
+#### Pre-requisites
+
+If you completed the environment cleanup steps at the end of the previous 
+workshop chapter your good to go! If you did not cleanup or ran into issues you 
+can run the following from inside the Kratix repo to get a fresh environment:
```

Comment:
I wonder if we need to provide a "test" for this rather than an assumption people are going step by step?

E.g. something like:
```suggestion
If you completed the environment cleanup steps at the end of the previous 
workshop chapter your good to go!

To verify, confirm the following command succeeds, but shows no promises:
```
kubectl --context kind-platform get promises
```

If necessary, you can use the following command to rebuild a fresh environment:
```

Reply:
maybe? I don't feel too strongly on this. I'd be inclined to leave as is, and then if we notice people are often starting the chapters without having cleaned up properly then we can add this check in?

Reply:
I am always up for waiting to see what is needed. How do you think we can find that out though if this is async? 🤔

Reply:
> How do you think we can find that out though if this is async?

thats a fair questions! I was thinking more about when we host the workshops, but its a good point that it will be hard to get the feedback for unhosted run throughs. Do you think we should cater more to unhosted?

Reply:
I _think_ that is our goal with these docs (in contrast to the ones we have now produced for KCDUK). But I am not sure I remember where that idea came from 😅

Reply:
lets chat through this when we do acceptance! a quick change to make if we decide to do so.